releases/release-1.26/release-notes/maps/pr-112732-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 112732
+releasenote:
+  text: 'kubeadm: support image repository format validation.'
```

Comment:
```suggestion
  text: 'kubeadm: supported image repository format validation.'
```

Reply:
Makes it really hard to read. I would rather change this to `kubeadm: now supports image repository format validation.`